opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -4,7 +4,7 @@
 This module provides various file manipulation skills for the OpenDevin agent.
 
 Functions:
-- open_file(path, line_number=None): Opens a file and optionally moves to a specific line.
+- open_file(path: str, line_number: int | None = 1, context_lines: int = 100): Opens a file and optionally moves to a specific line.
```

Comment:
Should we explain `context_lines`? It's not clear from just the definition.

Reply:
The docstring in the method itself does more explanation.
I thought to keep it shorter in the head was the intent?

Reply:
Fair enough. My main issue with this is that it's at an intermediate level of detail where there's enough detail to make the reader wonder what `context_lines` is, but not enough to tell them what it is. Possibly `context_lines` could even just be removed from this explanation.